src/main/java/uk/gov/companieshouse/api/accounts/transaction/Transaction.java

```diff
@@ -0,0 +1,152 @@
+package uk.gov.companieshouse.api.accounts.transaction;
+
+import java.util.Date;
+import java.util.Map;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Transaction {
+
+    private String id;
+
+    @JsonProperty("closed_at")
+    private Date closedAt;
+
+    @JsonProperty("company_number")
+    private String companyNumber;
+
+    @JsonProperty("created_at")
+    private Date createdAt;
+
+    private String kind;
+
+    private String reference;
+
+    private String status;
+
+    @JsonProperty("updated_at")
+    private Date updatedAt;
+
+    @JsonProperty("created_by")
+    private Map<String,String> createdBy;
```

Comment:
Same as above. Why is this a map?

Reply:
Not sure, this was copied from abridged, but this is the temporary code for now.